setup.py

```diff
@@ -34,7 +34,7 @@ def load_requirements(*requirements_paths):
 
 setup(
     name='edx-notifications',
-    version='0.8.3',
+    version='0.8.4',
```

Comment:
You're adding a new, backwards-compatible feature here, so please bump to 0.9.0, not 0.8.4.

Reply:
Sure :+1: